shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/EncryptAlterTableTokenGenerator.java

```diff
@@ -122,6 +126,17 @@ public boolean isGenerateSQLToken(final SQLStatementContext sqlStatementContext)
     
     private Collection<SQLToken> getAddColumnTokens(final String tableName, final String columnName,
                                                     final AddColumnDefinitionSegment addColumnDefinitionSegment, final ColumnDefinitionSegment columnDefinitionSegment) {
+        return containsConfigDataType(tableName, columnName) ? getAddColumnTokensWithConfigDataType(tableName, columnName, addColumnDefinitionSegment, columnDefinitionSegment) 
+                : getAddColumnTokensByDefault(tableName, columnName, addColumnDefinitionSegment, columnDefinitionSegment);
+    }
+    
+    private boolean containsConfigDataType(final String tableName, final String columnName) {
```

Comment:
@tuichenchuxin Can we move this method to EncryptRule? I think this logic is common, it may be used by other class.

Reply:
> @tuichenchuxin Can we move this method to EncryptRule? I think this logic is common, it may be used by other class.

Nice suggestion, Done.